test/mongoGetCommand.test.ts

```diff
@@ -107,18 +106,22 @@ suite("scrapbook parsing Tests", () => {
 
     test("second of two commands, Mac/Linux", () => {
         let line1 = "db.find()";
-        let line2 = "db.insertOne({'a':'b'})";
-        let text = `${line1}\n${line2}`;
-        let command = getCommandFromTextAtLocation(text, new Position(2, 0));
-        assert.equal(command.text, line2);
+        for (let i in [0, 1, 2]) {
+            let line2 = `db.insertOne({${wrapInQuotes("a", +i)}:'b'})`;
```

Comment:
Why the +?

Reply:
I was getting an error about i being a string and not a number. The simplest way to convert (knowing that the array contains numbers) was to `+`coerce it. 

![image](https://user-images.githubusercontent.com/30420973/42064227-4bc3a7fe-7aea-11e8-95d4-479f3c475075.png)

Reply:
for (let i: number in [0, 1, 2])
?

or just (for let i = 0; i <=2; ++i)

But not a big deal

Reply:
Typescript doesn't like the first one - no type annotation in a for ... in statement. 

I'll do the other one.